app/web/features/communities/locales/en.json

```diff
@@ -10,6 +10,18 @@
         "title": "Cancel event?",
         "message": "Are you sure you want to cancel this event?"
     },
+    "invite_community_dialog": {
+        "title": "Request a community invite for this event",
+        "message": "Use this form to request that the moderators send a notification about this event to all nearby users.",
+        "toast_success": "Thanks, the request has been submitted",
+        "link": "more info"
+    },
+    "invite_community_dialog_buttons": {
+        "open": "Invite the community",
+        "confirm": "Confirm",
+        "close": "Cancel"
```

Comment:
Are there generic versions of these two in the dictionary?

Reply:
nope, the only "confirm" string in the locale it's in the the hosting request formular